src/main.rs

```diff
@@ -201,6 +201,60 @@ async fn main() -> std::io::Result<()> {
         }
     });
 
+    // Reminding moderators to review projects which have been in the queue longer than 24hr
+    let pool_ref = pool.clone();
+    scheduler.run(std::time::Duration::from_secs(10 * 60), move || {
```

Comment:
So this is going to ping every 10 minutes? I.e., if something is left overnight for, let's say 8 hours of sleep, it will send 80 pings for a single project?

Reply:
yeah it does, will find a way to fix this